markdown.dtx

```diff
@@ -24182,6 +24390,574 @@ function M.reader.new(writer, options)
 % \par
 % \begin{markdown}
 %
+%#### Parsers Used for Indentation (local)
+%
+% \end{markdown}
+%  \begin{macrocode}
+
+  parsers.blockquote_start = parsers.leader * parsers.more * parsers.space^-1
+
+%  \end{macrocode}
+% \begin{markdown}
+%
+% Check if a trail exists and is non-empty in the indent table `indent_table.
```

Comment:
```suggestion
% Check if a trail exists and is non-empty in the indent table `indent_table`.
```

Reply:
What is a "trail"? This does not tell me much on top of what I can glean from the function and parameter names. It seems more useful to bundle all the `*_trail()` functions together and give a more general explanation of what is being done and why for all of them at the same time.

Reply:
I have added a short explanation in discussion #144 for now.